__tests__/server.test.js

```diff
@@ -52,3 +57,46 @@ describe("/api/topics", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  describe("GET request", () => {
+    test("200: should return the requested article to the client", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          const articleOne = {
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          };
+          expect(article).toEqual(articleOne);
```

Comment:
We should use to match object or object containing because if at a certain point we decide to add an extra property this test will break 🙂